README.md

```diff
@@ -50,7 +50,9 @@ Find more demo in [KVRawClientTest](https://github.com/birdstorm/KVRawClientTest
 
 See [Java Client Documents](/docs/README.md) for references about how to config and monitor Java Client.
 
-An [API reference](https://tikv.github.io/client-java/apidocs) is also available.
+A [Maven site](https://tikv.github.io/client-java/site) is also available. It includes:
+1. [API reference](https://tikv.github.io/client-java/site/xref/index.html) 
```

Comment:
This seems not an API reference, but a source code viewer. The API reference is missing now.

Reply:
OH! I am writing a patch adding a reporting plugin for API refs in the maven site.